### DISCUSSION TITLE
Loading bar Vs Spinners

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#499] [Proposals]
# Contexto
La loading bar es un componente que migramos de WebApp pero en ningún momento realmente ha pasado por un proceso de creación de componentes.

En la documentación de diseño no queda claro cuándo usar una loading bar vs un spinner.

- Tiene sentido tener la loading bar en Mística? Por qué?
- En qué casos usarías una loading bar en lugar de un spinner?

**Comment by aweell** [comment]
Reflejo aquí una síntesis de la conversación mantenida al respecto:

Uso de skeletons, spinners y loading bars


* **Skeleton**: 
  * Primera carga de contenido, usar para carga total de página 

* **Spinner**: 
  * Carga parcial de contenido desatada por acción del usuario.  
  * Carga de proceso dentro de un flujo.
  * Carga puente entre una pantalla de un producto y una webview externa.
  * Carga inline (dentro de componentes), el ejemplo del button loading.

Con respecto a los casos en los que se usaría una loading bar en mística. se abren dos incógnitas:

* **Loading bars**: 
  * Cargas en segundo plano?
  * Cargas de archivos?
